modules/local/fastq_dl/main.nf

```diff
@@ -0,0 +1,43 @@
+process FASTQ_DL {
+    tag "$id"
+    label 'process_medium'
+
+    conda "bioconda::fastq-dl=3.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fastq-dl:3.0.0--pyhdfd78af_0' :
+        'biocontainers/fastq-dl:3.0.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), val(id)
+    val(provider)
+
+    output:
+    tuple val(meta), path('reads/*.fastq.gz'), emit: reads
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    """
+    # Make directory to move final reads to
+    mkdir -p reads
+
+    fastq-dl \\
+        $args \\
+        --cpus $task.cpus \\
+        --provider $provider \\
+        --outdir ./ \\
+        --accession $id
+
+    # Move/rename reads
+    find * -type f   -name "$id*" -exec mv {} reads/${args2}{} \\;
```

Comment:
Do you have `find *` here so it's only running find on the output files from `fastq-dl` and skipping over any possible files under `reads/` (e.g., from a previous run)?

Another way might be to put all files from `fastq-dl` into `reads-initial/` directory, do a find only on files in that directory (`find reads-initial/ ...`) and move to `reads/` for the final set of files.

But what you have should work as well.

Reply:
This was following along with how the SRA module did it. I just had `find *` as I wanted to be able to add the prefix more easily when renaming the outputs

Reply:
Thanks. What you have is good and there's no need to change this.